test/functionalTest/cases/0501_cors/any_origin.test

```diff
@@ -0,0 +1,76 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+CORS any origin
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255 IPV4 -enableCORS __ALL
+
+--SHELL--
+echo "01. GET Entry Points"
+echo "=============================================================================="
+orionCurl --url /v2 -X GET
+echo
+echo
+
+echo "02. OPTIONS Entry Points"
+echo "======================================================"
+orionCurl --url /v2 -X OPTIONS
+echo
+echo
+
+--REGEXPECT--
+01. GET Entry Points
+==============================================================================
+HTTP/1.1 200 OK
+Content-Length: 135
+Access-Control-Allow-Origin: *
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "entities_url": "/v2/entities",
+    "registrations_url": "/v2/registrations",
+    "subscriptions_url": "/v2/subscriptions",
+    "types_url": "/v2/types"
+}
+
+
+02. OPTIONS Entry Points
+======================================================
+HTTP/1.1 200 OK
+Content-Length: 0
+Access-Control-Max-Age: REGEX([0-9]{5})
+Access-Control-Allow-Headers: REGEX(.*)
+Access-Control-Allow-Origin: *
+Access-Control-Allow-Methods: GET, OPTIONS
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB

```

Comment:
For completeness, please add a 'POST /v2' to see the correct 'bad verb' message

Reply:
Same as above.

Reply:
NTC